### DISCUSSION TITLE
Single language model other than English?

### DISCUSSION THREAD

**Discussion by ewwink** [openai/whisper#2082] [Q&A]
there is `medium.en` which is English only model, where are other single language models?
mainly I want Indonesian model.

**Comment by glangford** [comment]
There is a fine tuned Indonesian model over at Hugging Face.

https://huggingface.co/cahya/whisper-medium-id

> Whisper Medium Indonesian
This model is a fine-tuned version of [openai/whisper-medium](https://huggingface.co/openai/whisper-medium) on the Indonesian mozilla-foundation/common_voice_11_0, magic_data, titml and google/fleurs dataset.
